### PR TITLE
netbird: init at 0.8.6

### PR DESCRIPTION
**Pull request by dit7ya**
###### Description of changes

NetBird is an open-source VPN management platform built on top of WireGuard® making it easy to create secure private networks for your organization or home.

https://github.com/netbirdio/netbird

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/netbird/default.nix [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  vendorSha256 = "sha256-KtRQwrCBsOX7Jk9mKdDNOD7zfssADfBXCO1RPZbp5Aw=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/netbirdio/netbird/client/system.version=${version}"
+    "-X main.commit=${src.rev}"
+  ];
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/netbird/default.nix:42]: Why are tests disabled?

File: pkgs/tools/networking/netbird/default.nix [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    xorg.libXcursor
+    xorg.libXrandr
+    xorg.libXxf86vm
+    xorg.libXinerama
+    gtk3
+    libayatana-appindicator-gtk3
+  ];
+  nativeBuildInputs = [
+    pkg-config
+  ];
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/netbird/default.nix:31]:
```suggestion
  nativeBuildInputs = [
    pkg-config
  ];

  buildInputs = [
    xorg.libX11
    xorg.libXcursor
    xorg.libXrandr
    xorg.libXxf86vm
    xorg.libXinerama
    gtk3
    libayatana-appindicator-gtk3
  ];
```

File: pkgs/tools/networking/netbird/default.nix [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, xorg
+, gtk3
+, libayatana-appindicator-gtk3
+}:
+buildGoModule rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/netbird/default.nix:9]:
```suggestion
}:

buildGoModule rec {
```

**Comment by dit7ya** [comment]
Closing in favor of #186669.

**Pull request closed without merging**